### PR TITLE
Add route for activity details and link to comments.

### PR DESCRIPTION
**Pull request by manwithtwowatches**
https://www.openproject.org/work_packages/13648

### CODE REVIEW

File: public/templates/work_packages/tabs/_user_activity.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <div class="work-package-details-activities-activity-contents"
      ng-mouseover="showActions()"
      ng-mouseout="hideActions()">
-  <div class="comments-number"><a ng-href="#{{ currentAnchor }}" ng-bind="'#' + activityNo"></a>
+  <div class="comments-number">
+    <a name="{{ activityNo }}" ui-sref="work-packages.list.details.activity.details({ activity_no: activityNo })"
```
Comment (myabc) [inline public/templates/work_packages/tabs/_user_activity.html]: this isn't rendering with the activity_no

Comment (manwithtwowatches) [inline public/templates/work_packages/tabs/_user_activity.html]: @myabc seems to be working ok for me. come and give me a shout if you have a second to show me.

**Comment by hschink** [comment]
@manwithtwowatches The activity links work in the overview page but copying the link results in a `404` because the link copied is just malformed.

**Comment by manwithtwowatches** [comment]
@hschink The last commit gives a sensible link that can be copy pasted. For some reason ui-router doesn't like the ui-sref being on the <a>.

I'm aware that this doesn't go to the activity_no anchor though. I think this is because the activities haven't been loaded yet so will probably have to resort to another timeouty scrolling solution.

**Pull request merged**